### DISCUSSION TITLE
毕业了找工作好累

### DISCUSSION THREAD

**Discussion by 00zzxx** [share121/inter-knot#2520] [其他]
心累钱包也累。
怎么感觉我一个学期下来的花销都不如毕业这阵子的花费呢...光是入职体检就花了110，再加上一堆林林总总的下来居然花了230（老天我在学校一周早午饭也就花了100）<img src="https://bbs-static.miyoushe.com/static/2024/07/31/1a2616d760a6f334a4a9abca46267af3_7281603850910753959.png" alt="三月七 我" width="75" >算了，就当是为了那三千块钱的工资吧,,,,

**Comment by nhb-bcd** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
打工人现状

**Reply by 00zzxx** [reply]
<img src="https://bbs-static.miyoushe.com/static/2023/10/10/220bfd6cfc2a7180b0e05b964caf2eba_7913072307970350389.png" alt="阿库_流汗" width="75" >打工人是这样的
